includes/data-stores/class-wc-coupon-data-store-cpt.php

```diff
@@ -224,6 +224,9 @@ public function delete( &$coupon, $args = array() ) {
 	 * @since 3.0.0
 	 */
 	private function update_post_meta( &$coupon ) {
+		
+		$this->updated_props = array();
```

Comment:
Why this is needed? From the commit message, this change seems unrelated to this PR. If there is a separate you that is fixed by this, could you please open a separate PR?

Reply:
Yes, this is related to a new issue.
You can ignore it, I'll open a new pull request.